exercises/brandon_hewer/ex02_oo_basics/src/Circle.h

```diff
@@ -0,0 +1,52 @@
+#ifndef CIRCLE_H
+#define CIRCLE_H
+
+//----------------------------------------------------------------------
+// Includes
+//----------------------------------------------------------------------
+#include "Shape.h"
+
+#include <math.h>
+
+//===-- Circle.h - Shape definition -------*- C++ -*-=========================//
+//
+//                          2-Dimmensional Shapes
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file contains the declaration of the Circle class, which is used to
+/// initialize and access the data of circle shapes. Inherits from the Shape
+/// class.
+///
+//===----------------------------------------------------------------------===//
+class Circle :
+	public Shape
+{
+public:
+	/// Constructs a circle with a specified radius.
+	Circle(double radius);
+
+	/// Destructor.
+	~Circle();
+
+	/// Returns the perimeter of the circle.
+	double perimeter() const;
```

Comment:
please use the `override` specifier. See [here](http://en.cppreference.com/w/cpp/language/override). This is true for the other classes